### PR TITLE
Updated Prefab spawn and despawn tutorial for release

### PR DESCRIPTION
**Pull request by micronAMZN**
Updated tutorial, images, and movies for UX changes and feature updates. Made a bunch of style updates.

Signed-off-by: Mike Cronin <58789750+micronAMZN@users.noreply.github.com>

### CODE REVIEW

File: content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md [hunk 1]
```diff
@@ -6,110 +6,110 @@ weight: 300
...
+1. Connect the nodes. Drag from the **Out** pin on the **On Graph Start** node to the **Create Ticket** pin on the **Create Spawn Ticket** node to connect the nodes.
 
     {{< video src="/images/learning-guide/tutorials/entities-and-prefabs/connect-nodes.mp4" info="Click and drag to connect Script Canvas nodes." autoplay="true" loop="true" >}}
 
-1. Add the prefab reference to the Create Spawn Ticket node. On the Create Spawn Ticket node, **right-click** on the **Prefab** property and select **Convert to Reference** from the context menu. The property is automatically set to the **PrefabRef** variable you created in step **1**. You can choose the {{< icon settings.svg>}} settings icon in the **Prefab** field to select a variable reference if necessary.
+1. Add the prefab reference to the **Create Spawn Ticket** node. Right-click on the **Prefab** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **PrefabRef** variable you created in step **1**. You can choose the {{< icon settings.svg>}} **Settings** button in the **Prefab** field to select a variable reference if necessary.
```
Comment (chanmosq) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md:76]: Nit - remove bold from step numbers
```suggestion
1. Add the prefab reference to the **Create Spawn Ticket** node. Right-click on the **Prefab** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **PrefabRef** variable you created in step 1. You can choose the {{< icon settings.svg>}} **Settings** button in the **Prefab** field to select a variable reference if necessary.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md [hunk 2]
```diff
@@ -6,110 +6,110 @@ weight: 300
...
+    {{< image-width "/images/learning-guide/tutorials/entities-and-prefabs/connect-spawn-node.png" "900" "Connect the Spawn node to the Create Spawn Ticket node.">}}
 
-1. In addition to spawning prefab instances, the Spawn node also needs to populate the SpawnTicket variable with data. On the Spawn node, **right-click** on the **SpawnTicketOut** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **SpawnTicket** variable you created in step **2**. You can choose the {{< icon settings.svg>}} settings icon in the **SpawnTicketOut** field to select a variable reference if necessary.
+1. In addition to spawning prefab instances, the **Spawn** node also needs to populate the **SpawnTicket** variable with data. On the **Spawn** node, right-click on the **SpawnTicketOut** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **SpawnTicket** variable you created in step **2**. You can choose the {{< icon settings.svg>}} **Settings** button in the **SpawnTicketOut** field to select a variable reference if necessary.
```
Comment (chanmosq) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md:88]:
```suggestion
1. In addition to spawning prefab instances, the **Spawn** node also needs to populate the **SpawnTicket** variable with data. On the **Spawn** node, right-click on the **SpawnTicketOut** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **SpawnTicket** variable you created in step 2. You can choose the {{< icon settings.svg>}} **Settings** button in the **SpawnTicketOut** field to select a variable reference if necessary.
```

**Review by chanmosq — APPROVED** [review]
Nice! Approving with nit suggestion

**Review by sptramer — APPROVED** [review]
Approved pending Chanelle's requested changes. Looks good!

**Comment by chanmosq** [comment]
We may need to retarget this to main and merge after release?

**Pull request merged**